javaparser-core/src/main/java/com/github/javaparser/ast/nodeTypes/NodeWithJavadoc.java

```diff
@@ -64,7 +64,7 @@
      */
     @SuppressWarnings("unchecked")
     default N setJavadocComment(String comment) {
-        return setJavadocComment(new JavadocComment(comment));
+        return setJavadocComment(new JavadocComment(" " + comment));
```

Comment:
Why the space?

Reply:
My reasoning is: this is a helper method, my expectation would be to get a nice Javadoc. But I get a Javadoc where the text is stuck to the star:
```java
/**
 *Oh, I'm stuck!
 */
```
So I thought I would help the user. Of course, some people might be surprised for a bit because there are double spaces in the output, but I'm sure they'll manage ;-)

Reply:
Makes sense